.github/workflows/lint.yml

```diff
@@ -0,0 +1,41 @@
+name: Lint
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - main
+  # Replace pull_request with pull_request_target if you
+  # plan to use this action with forks, see the Limitations section
+  pull_request:
+    branches:
+      - main
+
+# Down scope as necessary via https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token
+permissions:
```

Comment:
Should these be `read`? I'm not sure I understand the docs.

Reply:
I just removed the permissions config altogether. It looks like it is around GITHUB_TOKEN permissions and we aren't using a token in this workflow.